pyvo/dal/tap.py

```diff
@@ -149,6 +149,38 @@ def tables(self):
                 vosi.parse_tables(response.raw.read), tables_url)
         return self._tables
 
+    def _parse_examples(self, examples_uri, /, depth=0):
```

Comment:
why do you make ``examples_uri`` positional only? I don't see any downside in spelling out argument names, and in cases of optional kwargs I in fact prefer them to be always spelt out.